src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -147,11 +147,37 @@ public void testSearchCrop() throws Exception {
         SearchRequest searchRequest =
                 new SearchRequest("and")
                         .setAttributesToCrop(new String[] {"overview"})
-                        .setCropLength(5);
+                        .setCropLength(1);
 
-        SearchResult searchResult = index.search(searchRequest);
+        Results res_gson = jsonGson.decode(index.rawSearch(searchRequest), Results.class);
```

Comment:
It is not common to define java variables with snake_case :)

Reply:
I know but all the others was this way so I let it the same. I will changes all of them